great_expectations/experimental/metric_repository/column_descriptive_metrics_metric_retriever.py

```diff
@@ -76,4 +82,71 @@ def _get_table_metrics(self, batch_request: BatchRequest) -> Sequence[Metric]:
                 exception=None,  # TODO: Pass through a MetricException() if an exception is thrown
             )
         )
+        # metric_name = "table.column_types"
+        # # TODO: fix this jankiness
```

Comment:
`table.column_types` has some unique behaviors that I wasn't able to get the bottom of. 

1. It has `include_nested=True` in the lookup key at times
2. It returns pandas types like `dtype(int64)` which caused some issues with the integration test.

Reply:
I was thinking we convert the types to strings for now since that is what we are displaying in the UI. We don't need to use this information for serialization, just for display.